src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/playback/PlaybackUtils.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2014 Ericsson
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+
+package com.sonyericsson.hudson.plugins.gerrit.trigger.playback;
+
+import java.io.File;
+
+import jenkins.model.Jenkins;
+import hudson.XmlFile;
+
+/**
+ * Utility class for Playback.
+ * @author scott.hebert@ericsson.com
+ */
+public final class PlaybackUtils {
+
+    /**
+     * Playback utils.
+     */
+    private PlaybackUtils() {
+    }
+
+    /**
+     * @return XmlFile corresponding to gerrit-trigger-server-timestamps.xml.
+     */
+    public static XmlFile getConfigXml() {
```

Comment:
Since this is the only method it could be moved into ```GerritMissedEventsPlaybackManager```.
But now I get why you thought of using a map, but you are risking each server instance overwriting this file so other servers will loose their data.

A directory with one file per server would be more safe I think.